filament/include/filament/VertexBuffer.h

```diff
@@ -116,6 +116,9 @@ class UTILS_PUBLIC VertexBuffer : public FilamentAPI {
          * @warning VertexAttribute::TANGENTS must be specified as a quaternion and is how normals
          *          are specified.
          *
+         * @warning Not all backends support 3-component attributes that are not floats. For help
```

Comment:
Do we have an API apps can query to know that?

Reply:
Mathias and I had discussed this, we wish to avoid adding too many queries.  One day in the future we might be able to do this conversion automatically.

Reply:
Yes so our current stance is that "vk doesn't support it" (even if some vk might).